### ISSUE TITLE
Example doesn't work.

### ISSUE THREAD

**Issue by afxalz** [ctu-mrs/octomap_mapping_planning#5]
It seems that mrs_pcl_tools launch files were modified.

**Comment by klaxalk** [comment]
thanks, working on it.

**Comment by klaxalk** [comment]
Hey, should be fixed now. Please, update and compile **simulation**, **uav_modules** and **octomap_mapping_planning**.

**Comment by klaxalk** [comment]
Sorry, there was still a problem. Now, it should work against the up-to-date uav_core and uav_modules.

**Comment by afxalz** [comment]
The update did not work because I built `PCL` from source in the `mrs_pcL_tools` pkg. It was built with `-march-native` flag which is removed from the new version of mrs pkgs. I did the following to clean the build-from-source.
1) run `sudo make unistall` in `mrs_pcl_tools/lib/pcl/build` directory.
2) run `sudo apt remove "*pcl-*"` to remove all pcl installations and links.
3) run the `mrs_pcl_tools/install/install.sh` script and say `n` when asked to build-from-source.
`catkin clean -bdy && catkin build ` should compile the pkgs now.
